apps/reference/docs/guides/api/generating-types.mdx

```diff
@@ -10,10 +10,10 @@ Supabase APIs are generated from your database, which means that we can use data
 
 The Supabase CLI is a single binary Go application that provides everything you need to setup a local development environment.
 
-You can [install the CLI](https://www.npmjs.com/package/supabase) via npm or other supported package managers.
+You can [install the CLI](https://www.npmjs.com/package/supabase) via npm or other supported package managers. The minimum required version of the CLI is [v1.8.1](https://github.com/supabase/cli/releases).
```

Comment:
Should this be pointing to our Docs rather than the repo @sweatybridge ?

I'll approve nonetheless so that you can push